Doc/includes/sqlite3/ctx_manager.py

```diff
@@ -14,3 +14,5 @@
         con.execute("insert into person(firstname) values (?)", ("Joe",))
 except sqlite3.IntegrityError:
     print("couldn't add Joe twice")
+
```

Comment:
Could you add a comment to make it explicit that we need to call the `close()` method explicitly to close the connection?